packages/next-templates/src/app/meldingen-openbare-ruimte/graffiti-melding/form/page.tsx

```diff
@@ -233,7 +233,7 @@ export default function Home() {
                     invalid={!!errors.name}
                     {...register('name', {
                       required: 'Dit is verplicht',
-                      minLength: { value: 4, message: 'Min length is 4' },
+                      minLength: { value: 3, message: 'Min length is 3' },
                     })}
                     placeholder=""
                     name="Naam"
```

Comment:
name wilt je in het engels want die naam gebruikt de backend. dus je wilt het consistent houden omdat de id en classname ook engels zijn.